Packages/com.unity.inputsystem/InputSystem/Utilities/ReadOnlyArray.cs

```diff
@@ -69,6 +69,12 @@ public TValue[] ToArray()
             return result;
         }
 
+        /// <summary>
+        /// Returns the index of the first element for which the given <paramref name="predicate"/> is true (if any).
+        /// </summary>
+        /// <param name="predicate">The unary predicate to be evaluated for each element.</param>
```

Comment:
Hmm.... since we are in C# predicate is also a concept so removed the unary (mathematical) aspect here and rewrote the description to make it clearer. Better?

Reply:
Not sure if you mean we should link to C# reference, e.g. https://docs.microsoft.com/en-us/dotnet/api/system.predicate-1?view=net-6.0? Or stay with current explanation? 

Added an example to the doc.